test/e2e/specs/wp-signup-spec.js

```diff
@@ -81,6 +81,10 @@ before( async function() {
 describe( `[${ host }] Sign Up  (${ screenSize }, ${ locale })`, function() {
 	this.timeout( mochaTimeOut );
 
+	beforeEach( async function() {
+		return await overrideABTests.setOverriddenABTests( driver, 'moveUserStepPosition', 'first' );
```

Comment:
I just found out that maybe we no longer need to set the default 

https://github.com/Automattic/wp-calypso/blob/master/client/lib/abtest/README.md#updating-ab-tests-in-the-e2e-tests

Unless you're switching between the two... ?

Reply:
Nice! I didn't realise it uses the default variation by ... _default_. I'll remove it :)

Reply:
Removed!